viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -74,10 +84,15 @@ using SensorId = cartographer::mapping::TrajectoryBuilderInterface::SensorId;
 const SensorId kRangeSensorId{SensorId::SensorType::RANGE, "range"};
 const SensorId kIMUSensorId{SensorId::SensorType::IMU, "imu"};
 
-// For a given color channel convert the scale from the given 102-255 range to
-// 100-0. This is an initial solution for extracting probability information
-// from cartographer
-unsigned char ViamColorToProbability(unsigned char color);
+// For a given color channel (probabilityColorChannel) convert the scale from
+// the given 102-255 range to 100-0. This is an initial solution for extracting
+// probability information from cartographer
+int calculateViamProbabilityFromColorChannel(
+    std::vector<unsigned char> pixel_data);
+
+// Check if pixel is not in our map and is the default color (i.e. RGB =
+// [102,102,102])
```

Comment:
[q] What does "Check if pixel is not in our map" mean? What does "default color" mean?

Reply:
"pixel not in our map" refers to the grey pixels that represent the background color. We want to filter out these pixels. Default color is `defaultCairosEmptyPaintedSlice`, more explicitly when all 3 color channels have the default value(resulting in the grey pixels) 

I changed the comment to better reflect this. Take a look!